core/src/avm2/globals/flash/events/GameInputEvent.as

```diff
@@ -1,4 +1,5 @@
 package flash.events {
+    [API("688")]
```

Comment:
I think this is AIR-only (well, according to the docs), so should be `689`?
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/GameInputEvent.html

But I feel like we've been here before, and this is one of the many cases where the docs just lie.